Cargo.toml

```diff
@@ -1,2 +1,7 @@
 [workspace]
 members = ["redis", "redis-test"]
+
+# Exclude ci directories, giving CI the ability to build
+# Packages without them being included in the workspace even thought they aren't
+# this one line has wasted 3 hours of my time
```

Comment:
I empathize with your pain here but can we remove the comment? :stuck_out_tongue:  

Actually, do we even need this? Would a .gitignore line for the `.ci` folder accomplish what's needed?

Reply:
No because the issue is that when I try to build redisjson for testing I do it in the .do folder and cargo goes up a couple levels and finds the workspace and complains about it being in the project but not being in the project

I'm happy to remove the comment though you're right.

I tried using a blob pattern to exclude the whole CI folder but it didn't work unfortunately

Reply:
Ok thanks for the explanation. I see the issue now. Can we not just move the directory where redis-json is checked out into a directory that isn't a subdir of the main project? 

Would really like to keep something that's purely CI-related out of Cargo.toml if possible.